tensorflow/core/lib/io/record_reader_writer_test.cc

```diff
@@ -158,6 +158,40 @@ TEST(RecordReaderWriterTest, TestBasics) {
   }
 }
 
+TEST(RecordReaderWriterTest, TestSkip) {
```

Comment:
Add tests for error cases, such as skipping more records than there are left and reading from a truncated file.